pkg/docker/config/config_test.go

```diff
@@ -225,6 +225,73 @@ func TestGetAuth(t *testing.T) {
 	}
 }
 
+func TestGetHelperAuth(t *testing.T) {
```

Comment:
This looks like something that should be fairly easy to add as one more test case in the existing `TestGetAuth`, without so much code duplication.

Reply:
I had some issues with adding it, turns out whatever is the last thing there will leave its config in the "other config path", which works with the current tests because of their layout.